tests/test_management.py

```diff
@@ -87,7 +86,7 @@ class TestIntegratedChannelCommandMixin(unittest.TestCase):
     Tests for the ``IntegratedChannelCommandMixin`` class.
     """
 
-    @ddt.data('SAP', 'DEGREED')
```

Comment:
[suggestion] Can't we replace Degreed with Degreed2 here and everywhere else in this file?

Reply:
I have replaced the removed Degreed configs with the Degreed v2.